src/cascadia/LocalTests_TerminalApp/SettingsTests.cpp

```diff
@@ -1191,4 +1193,36 @@ namespace TerminalAppLocalTests
             VERIFY_IS_TRUE(caughtExpectedException);
         }
     }
+    void SettingsTests::TestProfileIconWithEnvVar()
+    {
+        const std::string settingsJson{ R"(
+        {
+            "profiles": [
+                {
+                    "name": "profile0",
+                    "icon": "%WINDIR%\\System32\\x_80.png"
+                }
+            ]
+        })" };
+
+        VerifyParseSucceeded(settingsJson);
+        CascadiaSettings settings;
+        settings._ParseJsonString(settingsJson, false);
+    }
+    void SettingsTests::TestProfileBackgroundImageWithEnvVar()
+    {
+        const std::string settingsJson{ R"(
+        {
+            "profiles": [
+                {
+                    "name": "profile0",
+                    "backgroundImage": "%WINDIR%\\System32\\x_80.png"
+                }
+            ]
+        })" };
+
+        VerifyParseSucceeded(settingsJson);
+        CascadiaSettings settings;
+        settings._ParseJsonString(settingsJson, false);

```

Comment:
i like that you added tests, but you should validate that they actually did something! check the resulting values in the path variables.

since you can't really make sure WINDIR expanded to "the right value", you could "check whether there is a `%` in the final string" maybe?